NOTICE

```diff
@@ -0,0 +1,15 @@
+Puppet Module - puppetlabs-reboot
+
+Copyright 2013 - 2016 Puppet, Inc.
```

Comment:
Never guessed.  Based on 1st commit to the repo. https://github.com/puppetlabs/puppetlabs-reboot/commit/bb99c810e5a22c068828e365ca54123ea4e1c6cb